### PR TITLE
Cmek azure

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Review by AlexicaWright — COMMENTED** [review]
As a rule, we don't document third-party products. I'd recommend to rewrite this to running text to capture the information that pertains to Aura or anything Neo4j.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

**Comment by fiquick** [comment]
> As a rule, we don't document third-party products. I'd recommend to rewrite this to running text to capture the information that pertains to Aura or anything Neo4j.

Running text - does that mean more like blocks of text instead of numbering the bullet points? @AlexicaWright

**Comment by AlexicaWright** [comment]
Yes, exactly. I'll take this to Slack instead :)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

File: modules/ROOT/pages/platform/security/encryption.adoc [hunk 1]
```diff
@@ -99,4 +99,45 @@ In Aura, you can use AWS single-region keys, multi-region keys or replica keys a
...
+=== Creating an Azure key vault
+
+CAUTION: It's crucial to differentiate between Azure Key Vault and Azure Key Vault Managed HSM for your key management needs. 
+This document focuses on using *Azure Key Vault*.
+
+. In the Azure portal, create a link:https://learn.microsoft.com/en-us/azure/key-vault/general/[Key Vault]
+. In the *Basics tab* ensure the region matches your Aura database instance region and enable *purge protection*.
+. In the *Access configuration tab* enable *Azure role-based access control* and enable *Azure Disk Encryption for volume encryption*.
+. In the *Networking tab* allow access from all networks.
+. Select *Review + Create* and when the deployment is complete go to *resource* and record the *directory ID*, you need it later.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/encryption.adoc:115]:
```suggestion
[CAUTION]
====
It's crucial to differentiate between Azure Key Vault and Azure Key Vault Managed HSM for your key management needs.
This document focuses on using *Azure Key Vault*.
====

. Create a key vault in the Azure portal ensuring the region matches your Aura database instance region.
Enable *purge protection*, *Azure role-based access control*, and *Azure Disk Encryption for volume encryption*, and allow access from all networks.
. Once created and deployed, copy the *directory ID* from the *resource* tab, you need it later.
```

File: modules/ROOT/pages/platform/security/encryption.adoc [hunk 2]
```diff
@@ -99,4 +99,45 @@ In Aura, you can use AWS single-region keys, multi-region keys or replica keys a
...
+
+==== Add role assignment to the Key Vault
+
+. Go to *Access control (IAM)* and add *role assignment*.
+. Search *Key Vault Administrator* and tap *Key Vault Administrator* - this will highlight the row in grey, and then press *next*.
+. Select members (search by name, and when you have added the relevant person, press *select*) and then *Review + Assign*.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/encryption.adoc:121]:
```suggestion
=== Add role assignment to the Key Vault

In the Azure portal, add role assignment by selecting relevant members as *Key Vault Administrator*.
```

File: modules/ROOT/pages/platform/security/encryption.adoc [hunk 3]
```diff
@@ -99,4 +99,45 @@ In Aura, you can use AWS single-region keys, multi-region keys or replica keys a
...
+
+=== Creating a key
+
+Add a key in the Key Vault.
+When the key is created, record the *Key Identifier* because you need it later.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/encryption.adoc:126]:
```suggestion
When the key is created, record the *Key Identifier*, you need it later.
```

File: modules/ROOT/pages/platform/security/encryption.adoc [hunk 4]
```diff
@@ -99,4 +99,45 @@ In Aura, you can use AWS single-region keys, multi-region keys or replica keys a
...
+==== Create a Service Principal for the Neo4j CMK Application
+
+In the Azure tenant where your key is located, create a Service Principal linked to the Neo4j CMK Application with the ID displayed in the Aura Console.
+
+For example, using link:https://learn.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create-examples[Azure CLI] the command is:
+
+[source,bash]
+----
+`az ad sp create --id clientID`. 
+----
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/encryption.adoc:137]:
```suggestion
For example, using Azure CLI the command is:

[source,bash]
----
`az ad sp create --id clientID`.
----

See https://learn.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest#az-ad-sp-create-examples[Azure documentation^] for more information.
```

File: modules/ROOT/pages/platform/security/encryption.adoc [hunk 5]
```diff
@@ -99,4 +99,45 @@ In Aura, you can use AWS single-region keys, multi-region keys or replica keys a
...
+
+==== Add role assignment to the key
+
+. Go to *Access control (IAM)* and *Add role assignment.*
+. Select *Key Vault Crypto Officer* to highlight the row in grey, and then press *next*.
+. Tap *Select member* and assign access to *User, group, or service principal*, and paste the Neo4j CMK Application ID and add it to *Selected members*, then *Review + Assign*.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/encryption.adoc:143]:
```suggestion
In the Azure portal, add role assignment by selecting relevant members as *Key Vault Crypto Officer*.
Use the Neo4j CMK Application ID from the Aura Console.
```

**Review by AlexicaWright — COMMENTED** [review]
I've added suggestions that removes much details about Azure-specific things. But I haven't been able to test it out myself, so please just consider them as suggestions. If you do decide to accept, please ensure that you test and make sure I haven't omitted vital information or misunderstood.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-372.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-372.surge.sh/aura/platform/security/encryption)

**Comment by fiquick** [comment]
build broke so made a new PR

**Pull request closed without merging**